Guardfile

```diff
@@ -86,11 +86,11 @@ guard 'zeus' do
   # Ruby apps
   ruby = OpenStruct.new
   ruby.lib_files = /^(lib\/.+)\.rb$/
-
+=begin
```

Comment:
https://github.com/guard/guard-zeus says to just set `:rspec => false          # Don't use RSpec
`